controllers/applicationvolumereplication_controller.go

```diff
@@ -133,84 +212,23 @@ func (r *ApplicationVolumeReplicationReconciler) Reconcile(ctx context.Context,
 	}
```

Comment:
The Reconciler will be called for AVR creation, and from that point on, it will be recalled until it satisfies the app requirements, which is processing all subscriptions for an app successfully. If an app has no subscriptions, I believe that's an error and we should be treating it as such.